### DISCUSSION TITLE
Simple page with message to the user without logging in

### DISCUSSION THREAD

**Discussion by 0xf965** [filamentphp/filament#11370] [Help] [answered]
### Package

Panel builder

### Package Version

v3.1.2

### How can we help you?

After sending the verification email to the user, I want, upon clicking, to redirect to a SimplePage after entering the email verification controller, as documented in the Laravel documentation.

The problem is as follows: there are two types of pages: Page and SimplePage, both inherit from BasePage. Page allows getting a URL to redirect to, but it requires the user to have logged in, otherwise, it redirects to the login page. SimplePage does not have any URL.

The only way I have found to solve this is obviously through a page outside of Filament. But in that case, as far as I know, I need to rebuild the design.

The documentation does not mention SimplePage. How can we use them? Or solve the problem in another way?

**Comment by leandrocfe** [comment, marked as answer]
You can create a simple page without rebuilding the design.
- Just create a livewire component and extend the simple page

```php
namespace App\Livewire;

use Filament\Pages\SimplePage;

class CustomPage extends SimplePage
{
    protected static string $view = 'livewire.custom-page';
}
```
- Add this route in the routes/web.php

```php
use App\Livewire\CustomPage;
Route::get('/custom-page', CustomPage::class);
```
